tslint.json

```diff
@@ -1,140 +1,24 @@
-        "no-duplicate-variable": true,
-        "no-trailing-whitespace": true,
-        "no-unused-expression-chai": true,
-        "no-var-keyword": true,
-        "return-undefined": true,
-        "one-line": [true, "check-catch", "check-finally", "check-else", "check-open-brace", "check-whitespace"],
-        "prefer-const": true,
-        "promise-must-complete": true,
-        "semicolon": true,
-        "triple-equals": true,
-        "type-literal-delimiter": true,
-        "typedef": [true, "variable-declaration", "call-signature"],
-        "member-access": [
-            true,
-            "check-constructor"
-        ],
-        "no-irregular-whitespace": true,
-        "typedef-whitespace": [
-            true, {
-                "call-signature": "nospace",
-                "index-signature": "nospace",
-                "parameter": "nospace",
-                "property-declaration": "nospace",
-                "variable-declaration": "nospace"
-            }
-        ],
-        "no-relative-imports": {"severity": "off"},
-        "strict-boolean-expressions": false,
-        "max-classes-per-file": true,
-        "max-line-length": false,
-        "only-arrow-functions": {"options": ["allow-declarations"]},
-        "whitespace": [
-            true,
-            "check-module",
-            "check-rest-spread",
-            "check-type-operator",
-            "check-type-cast",
-            "check-branch",
-            "check-operator",
-            "check-separator",
-            "check-preblock",
-            "check-type",
-            "check-decl"],
-        "no-any": true,
-        "no-unused-expression": true,
-        "no-unnecessary-initializer": true,
-        "await-promise": [true, "Thenable"],
-        "completed-docs": false,
-        "no-unexternalized-strings": {"options": [{
-            "signatures": [
-                "localize",
-                "nls.localize"
-            ],
-            "keyIndex": 0,
-            "messageIndex": 1
-        }]},
-        "linebreak-style": false,
-        "quotemark": false,
-        "newline-per-chained-call": false,
-        "member-ordering": false,
-        "no-parameter-properties": false,
-        "no-inferrable-types": false,
-        "strict-type-predicates": true,
-        "increment-decrement": [true, "allow-post"],
-        "no-implicit-dependencies": false,
-        "ordered-imports": false,
-        "variable-name": false,
-        "prefer-readonly": false,
-        "comment-format": false,
-        "align": false,
-        "no-import-side-effect": false,
-        "switch-default": false,
-        "trailing-comma": false,
-        "switch-final-break": false,
-        "strict-string-expressions": false,
-        "newline-before-return": false,
-        "object-literal-sort-keys": false,
-        "no-suspicious-comment": false,
-        "export-name": false,
-        "no-magic-numbers": false,
-        "no-unsafe-any": false,
-        "no-unbound-method": false,
```

Comment:
Wow.

Reply:
Upgraded to the non-deprecated `typescript-eslint` and enabled the `recommended` ruleset with many fewer exceptions than before this PR.

---
In reply to: [432157363](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r432157363) [](ancestors = 432157363)